<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCloudMusicApi Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="TabCommon" name="tabCommon">
       <attribute name="title">
        <string>Common</string>
       </attribute>
      </widget>
      <widget class="TabApiTest" name="tabApiTest">
       <attribute name="title">
        <string>api_test</string>
       </attribute>
      </widget>
      <widget class="TabApiTest" name="tabEapiTest">
       <attribute name="title">
        <string>eapi_test</string>
       </attribute>
      </widget>
      <widget class="TabApiTest" name="tabWeapiTest">
       <attribute name="title">
        <string>weapi_test</string>
       </attribute>
      </widget>
      <widget class="TabAes" name="tabAes">
       <attribute name="title">
        <string>AES</string>
       </attribute>
      </widget>
      <widget class="TabRsa" name="tabRsa">
       <attribute name="title">
        <string>RSA</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabAes</class>
   <extends>QWidget</extends>
   <header>tabaes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabRsa</class>
   <extends>QWidget</extends>
   <header>tabrsa.h</header>
  </customwidget>
  <customwidget>
   <class>TabApiTest</class>
   <extends>QWidget</extends>
   <header>tabapitest.h</header>
  </customwidget>
  <customwidget>
   <class>TabCommon</class>
   <extends>QWidget</extends>
   <header>tabcommon.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
